src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareExploration.kt

```diff
@@ -70,7 +71,11 @@ class HardwareExploration(
         val awsExecutor = Executors.newFixedThreadPool(awsParallelism)
         val explorationExecutor = Executors.newFixedThreadPool(space.size)
         try {
-            return exploreHardwareInParallel(space, explorationExecutor, awsExecutor)
+            val result = exploreHardwareInParallel(space, explorationExecutor, awsExecutor)
+            if(!hardwaresWithError.isEmpty()){
```

Comment:
Code style - I recommend reformatting via IntelliJ all the time

Reply:
sure